packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -3471,6 +3471,13 @@ abstract class Element extends DiagnosticableTree implements BuildContext {
            0;
   }
 
+  /// Setting this property, the [debugVisitOnstageChildren] method will continue to
+  /// visit potentially onstage children for elements that are offstage, which may
+  /// introduce some additional overhead.
+  ///
+  /// By default, this property is set to false.
+  static bool debugVisitOffstageChildren = false;
```

Comment:
should this default to true, or this is meant to be a migration flag?

Reply:
naming nit: maybe `debugFinderVisitOffstageChildren` or some such? debugVisitOffstageChildren sounds like it affects the behavior of the `visitChildren` method.

Reply:
By default, setting it to `false` can avoid unnecessary recursion of offstage elements. It only needs to be set to `true` when we need to test things like an offstage `OverlayPortal`. I have added some documentation to explain this.